environment/os/install_deps.sh

```diff
@@ -18,6 +36,7 @@ Usage:
     ./install_deps.sh [command] [dependency_set]
 
 Commands:
+    prepare       Downloads and extracts specified dependencies.
```

Comment:
Add info that it can only be run with `TOOLCHAIN_RUN_DEPS`

Reply:
Done ✅
Added clarification to command help that only TOOLCHAIN_RUN_DEPS is supported for prepare